.ci/azure-pipelines.yml

```diff
@@ -78,15 +78,21 @@ jobs:
         env:
           BINTRAY_KEY: $(bintray_key)
 
+      - task: CopyFiles@2
+        displayName: 'Copy build assets'
+        inputs:
+          Contents: |
+            **/distributions/*-$(JELLYFIN_VERSION).zip
+            **/libs/*-$(JELLYFIN_VERSION).jar
+            **/libs/*-$(JELLYFIN_VERSION)-sources.jar
```

Comment:
I just realized we don't include the platform-android .aar files. They are in the `./jellyfin-platform-android/build/outputs/aar/` directory and use `jellyfin-platform-android-debug.aar` and `jellyfin-platform-android-release.aar` as file names. We should to rename them (probably Gradle files) to use the version in the filename.

_Although this might be out of scope for this PR_

Reply:
I have to sadly admit, that my skills in gradle scripting are lacking, so I don't feel comfortable editing those.
However, I can add a content search mask for the aar files within this PR if you want me to. :wink: